src/main.ts

```diff
@@ -1304,13 +1325,13 @@ function generateFromJson(ctx: Context, fullName: string, fullTypeName: string,
       } else if (isValueType(ctx, field)) {
         const valueType = valueTypeName(ctx, field.typeName)!;
         if (isLongValueType(field) && options.forceLong === LongOption.LONG) {
-          return code`${capitalize(valueType.toCodeString())}.fromValue(${from})`;
+          return code`${capitalize(valueType.toCodeString([]))}.fromValue(${from})`;
         } else if (isLongValueType(field) && options.forceLong === LongOption.BIGINT) {
           return code`BigInt(${from})`;
         } else if (isBytesValueType(field)) {
-          return code`new ${capitalize(valueType.toCodeString())}(${from})`;
+          return code`new ${capitalize(valueType.toCodeString([]))}(${from})`;
```

Comment:
Huh, are these changes required?

Reply:
These are calls to an internal API that  shouldn' t be used. The API has changed, I tried to track down where the error originated from (how I got a different dependency), but after some trying I decided to give the methods the array they needed.